near_api/providers.py

```diff
@@ -31,9 +34,9 @@ def json_rpc(self, method: str, params, timeout=2) -> dict:
             raise JsonProviderError(content["error"])
         return content["result"]
 
-    def send_tx(self, signed_tx: bytes) -> dict:
+    def send_tx(self, signed_tx: bytes, timeout: int=2) -> dict:
```

Comment:
I would use `float` instead of `int`, and [requests](https://requests.readthedocs.io/en/latest/api/#requests.request) even allows to pass a tuple of floats for separate connection timeout and read timeout

P.S. It makes sense to set the type on the `json_rpc` method as well.

Reply:
So would you recommend using a type alias like
`TimeoutType = Union[float, Tuple[float, float]]`
and use it as the type hint?

Reply:
Yep, that would be ideal, and leave the comment what those floats mean (at least copy from requests documentation) with the reference to requests documentation for future reference

Reply:
Ok. I pushed again. What do you think?